contents/uix/accessibility.md

```diff
@@ -0,0 +1,172 @@
+
+Such disabilities might affect website usage. It could be due to difficulty in understanding how to complete a task, remembering how to do something that was previously accomplished, increased frustration at confusing workflows or inconsistent layouts/navigation/other page features.
+
+Unlike other web accessibility issues, there is no quick fix to issues arising from cognitive disabilities. The rule of thumb you can follow is to always design your websites to be as logical, consistent, and usable as possible. Here is a checklist that you can follow:
+
+* Avoid distractions: <br>
+People with cognitive disabilities are more easily distracted, or they may have heightened sensory awareness where busy pages can be overwhelming.
+  * Avoid movement unless the moving elements can be frozen by the user
+  * Have a clean and uncluttered design
+  
+  Here's a comparison of a cluttered vs clean website landing page taken from British Gas website.
+  
+  Cluttered page:
+  ![Cluttered landing page](accessibility/bg-old.png "Cluttered landing page")
+
+  Clean page:
+  ![Clean landing page](accessibility/bg-new.png "Clean landing page")
+
+* Be logical:
+  * Multi-stage processes are broken down into logical steps, with regular reminders of how far through the process you are, and how long you've got left to complete the process, if appropriate.
+  * Workflows are logical, simple, and require as few interactions as possible to complete.
+
+* Be consistent:
+  * Navigation, header, footer, and main content are always in the same places
+  * Navigation mechanisms are consistent in appearance and behaviour
+
+* Communicate well: <br>
+People with cognitive disabilities are more likely to take things literally and to benefit from more visual cues.
+  * Pages are not overly long or dense in terms of the amount of information presented at once.
+  * The language used is as plain and easy to follow, and not full of unneccessary jargon and slang.
+  * Important points and content are highlighted in some way.
+  * User errors are clearly highlighted, with help messages to suggest solutions.
+
+> Taken from [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility#People_with_cognitive_impairments), [National Autistic Society](https://www.autism.org.uk/professionals/others/website-design.aspx)
+
+If you feel that the checklist above is rather obvious, it is because they are good design practices in general. More importantly, after following the checklist, you should do user testing with people with cognitive disabilities. Sometimes the web page may look fine to someone without disabilities, but it could be difficult for someone with autism or ADHD to navigate through the web page.
```

Comment:
> If you feel that the checklist above is rather obvious, it is because they are good design practices in general.

This sentence seems a little uncalled for?